base/docs/basedocs.jl

```diff
@@ -1816,10 +1846,18 @@ Symbol(x...)
 
 Construct a tuple of the given objects.
 
+See also [`Tuple`](@ref), [`NamedTuple`](@ref).
+
 # Examples
 ```jldoctest
-julia> tuple(1, 'a', pi)
-(1, 'a', π)
+julia> tuple(1, 'b', pi)
+(1, 'b', π)
+
+julia> ans === (1, 'b', π)
+true
+
+julia> Tuple([1, 2, pi])  # takes a collection
+(1.0, 2.0, 3.141592653589793)
```

Comment:
I think this may be confusing, because of the promotion happening in the array.
```suggestion
julia> Tuple(Real[1, 2, pi])  # takes a collection
(1, 2, π)
```